libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/offline/UnpackerTask.java

```diff
@@ -0,0 +1,33 @@
+package com.mapbox.services.android.navigation.v5.navigation.offline;
+
+import android.os.AsyncTask;
+
+import com.mapbox.navigator.Navigator;
+import com.mapbox.services.android.navigation.v5.navigation.NavigationLibraryLoader;
+
+import java.io.File;
+
+
+/**
+ * Takes in a string for a path to a TAR file containing routing tiles, and unpacks them to the
+ * specified destination path. The path to the TAR file containing routing tiles and the path to
+ * the directory in which to unpack the tiles are included in the params passed to this AsyncTask.
+ */
+public class UnpackerTask extends AsyncTask<String, Integer, File> {
+
+  static {
+    NavigationLibraryLoader.load();
+  }
+
+  @Override
+  protected File doInBackground(String... strings) {
+    Navigator navigator = new Navigator();
+    navigator.unpackTiles(strings[0], strings[1]);
```

Comment:
@kevinkreiser does creating a new `Navigator` here for unpacking the tiles create a disconnect with the initialization and offline route fetching that will happen in the future with a different instance of `Navigator`?